run.sh

```diff
@@ -29,39 +45,78 @@ case "${RUNNER}" in
     ;;
 esac
 
+# Use the user defined container, else the default container.
+IMAGE=${CHEDOCSIMAGE:-quay.io/eclipse/che-docs}
+
+# Default mountmoint for the container
 SRC_PATH="$(pwd)/src/main"
 
-$RUNNER pull $IMAGE
+# Define the run command as a function (shellcheck recommendation)
+runner() {
+  # Pull the default container image, else assume the image is present.
+  [ "${IMAGE}" = "quay.io/eclipse/che-docs" ] && "${RUNNER}" pull "${IMAGE}"
```

Comment:
@themr0c, this will pull a new img even if you already have one downloaded. One would need to re-tag the custom img manually in order to have it used.

Reply:
If your local image is called, let's say `che-docs`, then the pull is skipped. If it's the standard image  `quay.io/eclipse/che-docs`, then the pull is not skipped. Wasn't it the request?